Gemfile

```diff
@@ -36,7 +36,12 @@ gem 'jbuilder', '~> 2.5'
 # Reduces boot times through caching; required in config/boot.rb
 gem 'bootsnap', '>= 1.1.0', require: false
 
+# Authentication
 gem 'devise', '~> 4.6.2'
+gem 'devise_invitable', '~> 2.0.0'
+
+# Authorization
+gem 'pundit'
```

Comment:
:tada:

Reply:
It was already in here, I just moved it up!